### PR TITLE
reg.drone Battery parameters add nominal voltage

### PR DESCRIPTION
**Pull request by PetervdPerk-NXP**
As discussed in #112 
Adds the nominal voltage (the voltage that's typically on battery pack) to the reg.drone.battery.parameters message.

Battery Pack

```
uavcan.si.unit.voltage.Scalar.1.0 nominal_voltage
# The average voltage(not per-cell) the battery outputs when charged.
```

Deprecates Parameters.0.2

### CODE REVIEW

**Review by pavel-kirienko — APPROVED** [review]

**Comment by echoGee** [comment]
To be picky about this , it may be a bit more clear to mention the voltage at 50% SOC ? The voltage drop is not linear from 100-0% SOC and a defining a midpoint at SOC may be more accurate, than (max-min)/2.

**Comment by PetervdPerk-NXP** [comment]
But voltage at 50% State of Charge isn't the nominal voltage.

For example LiPo batteries don't have a linear discharge curve, the nominal voltage is the average voltage from this curve for example from 4.2V (Full) to 3.0V (Critical) and then you the calculate the average of the area. 

Which for LiPo is typically 3.7V.

I don't have a LiPo discharge plot at hand, but this source gives you a nice picture 

afbeelding
Source: https://www.labatteria.co.il/en/e-bike-battery-discharge-rate-4c-vs-3c/

**Comment by echoGee** [comment]
> But voltage at 50% State of Charge isn't the nominal voltage.
> 
> For example LiPo batteries don't have a linear discharge curve, the nominal voltage is the average voltage from this curve for example from 4.2V (Full) to 3.0V (Critical) and then you the calculate the average of the area.
> 
> Which for LiPo is typically 3.7V.
> 
> I don't have a LiPo discharge plot at hand, but this source gives you a nice picture
> 
> afbeelding
> Source: https://www.labatteria.co.il/en/e-bike-battery-discharge-rate-4c-vs-3c/

@PetervdPerk-NXP : Is "nominal voltage is the integral under the 1C discharge curve" an accepted definition? I couldn't find that definition. Also, do you mean integral divided by the capacity ?

File: reg/drone/service/battery/Parameters.0.3.uavcan
Comment (pavel-kirienko) [inline reg/drone/service/battery/Parameters.0.3.uavcan:68]: Suggestion based on the conversation at #112

```suggestion
# The nominal voltage of the battery pack (not per-cell) as defined by the vendor.
# The Smart Battery Data Specification v1.1 uses the term "design voltage".
# E.g., a typical 22S LiCoO2 pack might report 81.4 V here.
```

Comment (echoGee) [reply reg/drone/service/battery/Parameters.0.3.uavcan:68]: This, IMHO, is not a good way to define a standard parameter. lack of clarity.

Comment (pavel-kirienko) [reply reg/drone/service/battery/Parameters.0.3.uavcan:68]: While we're waiting for the SBS Forum to respond, we could merge this tentatively to unblock @PetervdPerk-NXP and then go back and provide a proper definition later.

Comment (echoGee) [reply reg/drone/service/battery/Parameters.0.3.uavcan:68]: Not sure I understand. Why would a nominal voltage (the definition of which is not known) block @PetervdPerk-NXP ?

Comment (pavel-kirienko) [reply reg/drone/service/battery/Parameters.0.3.uavcan:68]: This is discussed in the beginning of #112

**Pull request merged**
